Framework/Algorithms/src/MergeRuns/SampleLogsBehaviour.cpp

```diff
@@ -396,19 +458,46 @@ void SampleLogsBehaviour::checkErrorProperty(
     const MatrixWorkspace_sptr &addeeWS, Property *addeeWSProperty,
     const SampleLogBehaviour &behaviour, const double addeeWSNumber,
     const double outWSNumber, const std::string name) {
+
+  if (!isWithinTolerance(behaviour, addeeWSNumber, outWSNumber) &&
+      !stringPropertiesMatch(behaviour, addeeWSProperty)) {
```

Comment:
Yes, it is actually a bit different, although it should not change anything. It might be a little safer as if `isWithingTolerance` returns false, but the strings match, the warning or error will not be generated, though that should never happen.